docs/tutorial/ecommerce-tutorial/index.md

```diff
@@ -171,6 +171,10 @@ const Checkout = class extends React.Component {
 export default Checkout
 ```
 
+Be sure to enable the "Checkout client-only integration" on your Stripe dashboard to be able to use the redirectToCheckout integration. You can activate the integration in your [Stripe settings](https://dashboard.stripe.com/account/checkout/settings).
```

Comment:
```suggestion
Be sure to enable the "Checkout client-only integration" on your Stripe dashboard to be able to use the `redirectToCheckout` integration. You can activate the integration in your [Stripe settings](https://dashboard.stripe.com/account/checkout/settings).
```

Reply:
You use double in this line and others - and I can not comment on lines not in PR changes:

Can you also change from single quotes to double quotes in line 117

> `'Viewing test data'`